packages/server/src/arpa_reporter/lib/audit-report.js

```diff
@@ -429,10 +429,13 @@ function createHeadersProjectSummariesV2(projectSummaryGroupedByProject) {
     return headers;
 }
 
-async function generate(requestHost, tenantId) {
+async function generate(requestHost, tenantId, periodId = null) {
```

Comment:
```suggestion
async function generate(requestHost, tenantId, periodId) {
```

What's the purpose of transforming this default to `null` instead of `undefined`?